exercises/forth/canonical-data.json

```diff
@@ -0,0 +1,219 @@
+{
+  "#": [
+    "The cases are split into multiple sections, all with the same structure.",
+    "In all cases, the `expected` key is the string representation of the resulting stack",
```

Comment:
for completeness, should clarify: "the resulting stack after executing the Forth program contained in the `input`" or something to that effect.